Doc/library/http.client.rst

```diff
@@ -115,6 +115,25 @@ The module provides the following classes:
       The *strict* parameter was removed. HTTP 0.9 style "Simple Responses" are
       no longer supported.
 
+This module provides the following function:
+
+.. function:: parse_headers(fp)
+
+   Parse the headers from a file pointer *fp* representing a HTTP
+   request/response. The file has to be a :class:`BufferedIOBase` reader
+   (i.e. not text) and must provide a valid :rfc:`2822` style header.
+
+   This function returns an instance of :class:`http.client.HTTPMessage`
+   that holds the header fields, but no payload
+   (the same as :attr:`HTTPResponse.msg`
+   and :attr:`http.server.BaseHTTPRequestHandler.headers`).
+   After returning, the file pointer *fp* is ready to read the HTTP body.
+
+   .. note::
+      :meth:`parse_headers` does not parse the start-line of a HTTP message;
+      it only parses the `Name: value` lines. The file has to be ready to
```

Comment:
Ah! The Travis-CI build is failing due to the `Name: value` being wrapped in single backticks. If you make those double backticks this will pass and then we can merge.

```
$ make check
python3 tools/rstlint.py -i tools -i ./venv -i README.rst
[2] library/http.client.rst:134: default role used
1 problem with severity 2 found.
make: *** [check] Error 1
```

Reply:
Hi @epicfaace  - this is a minor change request, and once this is addressed, the PR can be merged.

Reply:
done!

Reply:
The change you made was to the wrong part. parse_headers was fine and should have single backticks, but on line 134 the "Name: value" should be wrapped in double ticks and then it looks like the build will pass fine.